flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/status/JobStatus.java

```diff
@@ -24,16 +24,25 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-/** Status of an individual job within the Flink deployment. */
+/** Last observed status of the Flink job within an application deployment. */
 @Experimental
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
 public class JobStatus {
+    /** Name of the job. */
     private String jobName;
+
+    /** Flink JobId of the Job. */
     private String jobId;
+
+    /** Last observed state of the job. */
     private String state;
+
+    /** Start time of the job. */
     private String updateTime;
```

Comment:
Maybe we could add a new filed `startTime`. IIUC, updating the `updateTime` every time will not cause busy reconciliation now.

Reply:
Could you please open a ticket for this?

Reply:
I will do this.